src/main/java/eu/beezig/core/api/SettingInfo.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of BeezigForge.
+ *
+ * BeezigForge is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * BeezigForge is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with BeezigForge.  If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
I'm sure you can remove one licence block :upside_down_face: 
Moreover, I'd add a `Copyright (C) 2020 Beezig Team` line or similar.

Reply:
Fixed